independent-projects/tools/devtools-common/src/main/resources/codestarts/quarkus-extension/code/quarkiverse/java/LICENCE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Rename this file to `LICENSE` (with an `S`)

Reply:
Oupsy :)

Reply:
licence is french 😅 I always mix them up